src/ReportPortal.SpecFlowPlugin/packages.config

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
-  <package id="ReportPortal.Client" version="1.2.3" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="10.0.2" targetFramework="net45" />
```

Comment:
I would keep Newtonsoft.Json at the same major version as ReportPortal.Client to avoid dependency issues.